src/Explorer/Controls/Settings/SettingsComponent.tsx

```diff
@@ -1000,15 +1000,18 @@ export class SettingsComponent extends React.Component<SettingsComponentProps, S
         tab: SettingsV2TabTypes.IndexingPolicyTab,
         content: <IndexingPolicyComponent {...indexingPolicyComponentProps} />
       });
-    } else if (
-      this.container.isMongoIndexEditorEnabled() &&
-      this.container.isPreferredApiMongoDB() &&
-      this.container.isEnableMongoCapabilityPresent()
-    ) {
-      tabs.push({
-        tab: SettingsV2TabTypes.IndexingPolicyTab,
-        content: <MongoIndexingPolicyComponent {...mongoIndexingPolicyComponentProps} />
-      });
+    } else if (this.container.isPreferredApiMongoDB()) {
+      if (isEmpty(this.container.features())) {
```

Comment:
>if (isEmpty(this.container.features())) { [](start = 6, length = 41)

how does the user get into this?

Reply:
user does not. this is a check for the hosted explorer which does not get any portal features. In that case, We are displaying the error that asks user to login through portal if they want to use the mongo index editor.